### DISCUSSION TITLE
歷史記錄不產生.md檔

### DISCUSSION THREAD

**Discussion by ixqkedp** [GaiZhenbiao/ChuanhuChatGPT#949] [General]
請問歷史記錄如何設定成只產生json檔就好，不要產生.md檔
或者是在刪除畫面左邊的歷史紀錄時也同時刪除.md檔

**Comment by Keldos-Li** [comment]
大概是我们忘了做删除/重命名json时同时操作md文件了。

您可以开启一个新的 issue (bug) 提交该问题。
